pkg-manager/core/package.json

```diff
@@ -144,9 +144,11 @@
     "registry-mock": "registry-mock",
     "test:jest": "jest",
     "test:e2e": "registry-mock prepare && run-p -r registry-mock test:jest",
+    "filtered-test:jest": "sh -o nounset -c 'jest --verbose --testPathPattern \"$TEST_PATH_PATTERN\" --testNamePattern \"$TEST_NAME_PATTERN\"'",
```

Comment:
How would this work? The registry-mock is not started.

Reply:
So that why `successful offline installation` test didn't finish!

Anyway, with the latest commits, I have merged `filtered-test` into `test`. The environment variables have been made optional.

Reply:
Also, since I merge `filterered-test` into `test`, `test` now requires `sh` even on Windows. Do you prefer creating a JavaScript file to spawn the test cross-platform?